kasa/modules/__init__.py

```diff
@@ -0,0 +1,78 @@
+"""Sub Package for device family independant modules."""
+
+# Iot Modules
```

Comment:
I think the question here is, if we should just export the common modules here? Because otherwise, we would be making all those implementations part of the public API.

Reply:
Exposed `iot` specific with a `Iot` prefix.  See more detailed comment below.